aquarius/app/assets.py

```diff
@@ -461,6 +465,84 @@ def update(did):
         return f'Some error: {str(err)}', 500
 
 
+@assets.route('/ddo/transferownership/<did>', methods=['PUT'])
+def transferownership(did):
```

Comment:
we have to follow the same function naming convention (use underscores).

Reply:
fixed